access/mattermost/mattermost_test.go

```diff
@@ -27,6 +28,8 @@ import (
 )
 
 var msgFieldRegexp = regexp.MustCompile(`(?im)^\*\*([a-zA-Z ]+)\*\*:\ +(.+)$`)
+var requestReasonRegexp = regexp.MustCompile(`(?im)^\*\*Reason\*\*:\ ` + "```\\n" + `(.*?)` + "```" + `(.*?)$`)
+var resolutionReasonRegexp = regexp.MustCompile(`(?im)^\*\*Resolution reason\*\*:\ ` + "```\\n" + `(.*?)` + "```" + `(.*?)$`)
```

Comment:
nit: same as above
```suggestion
var requestReasonRegexp = regexp.MustCompile("(?im)^\\*\\*Resolution Reason\\*\\*:\\ ```\\n(.*?)```(.*?)$")"
```